Applications/DataExplorer/mainwindow.cpp

```diff
@@ -857,7 +858,7 @@ void MainWindow::mapGeometry(const std::string &geo_name)
     else // use mesh from ProjectData
         mesh = _project.getMeshObjects()[choice-2].get();
 
-    std::string const& new_geo_name = dlg.getNewGeoName();
+    std::string new_geo_name = dlg.getNewGeoName();
```

Comment:
Just a note: Better to keep this one const and make another one in lines 870s to clarify the dependencies of the names...